### PR TITLE
DRAFT Add support for custom entrypoing

### PR DESCRIPTION
**Pull request by victorpoluceno**

### CODE REVIEW

File: charts/cf-review-env/templates/deployment.yaml [hunk 1]
```diff
@@ -31,6 +31,9 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy | default "IfNotPresent" }}
+          {{- if .Values.customCommand }}{{ if eq .Values.customCommand.enabled true }}
```
Comment (rodrigo-findhotel) [inline charts/cf-review-env/templates/deployment.yaml:34]: @victorpoluceno, I believe that you can use this one:

```
{{- if eq .Values.customCommand.enabled true }}
   command: ["{{ .Values.customCommand.command }}"]
{{- end }}
```

**Review by rodrigo-findhotel — APPROVED** [review]
I left a small comment, but LGTM!

File: charts/cf-review-env/templates/deployment.yaml (hunk 1 above)
Comment (victorpoluceno) [reply charts/cf-review-env/templates/deployment.yaml:34]: Yeah, that is actually a doubt that I have. What will happen if `customCommand` is not present, won't it break? that is why I added the first if.

Comment (rodrigo-findhotel) [reply charts/cf-review-env/templates/deployment.yaml:34]: I made a local test and with two formats it works for me, I just rendered the template locally, I dind't install it, but if I were you I would use the format that we are using in the other customizations just to keep conformity. =)

Comment (victorpoluceno) [reply charts/cf-review-env/templates/deployment.yaml]: @rodrigo-findhotel this is the error that I get when running the review env without customCommand in the values file: 

```Error: UPGRADE FAILED: template: cf-review-env/templates/deployment.yaml:34:27: executing "cf-review-env/template
s/deployment.yaml" at <.Values.customCommand.enabled>: nil pointer evaluating interface {}.enabled```

That is why the dind block also had to be add to all values file, which is not what we want. So, I'm going to to add that first if back.

Comment (rodrigo-findhotel) [reply charts/cf-review-env/templates/deployment.yaml]: Hmm, I see in! In that case, I can update our templates with this treatment and then clean the values files.

**Pull request merged**